Woof/Woof/ViewModel/LoginViewModel.swift

```diff
@@ -4,25 +4,20 @@ import Foundation
 final class LoginViewModel: ObservableObject {
     // MARK: - Internal interface
 
-    /// Indicates whether the owner role is selected.
-    ///
-    /// Returns `true` if the owner role is selected, otherwise `false`.
-    ///
-    /// The property publishes updates when the value is changed.
-    var isOwnerRoleSelected = false {
-        didSet {
-            if !isCurrentOwnerExists {
-                saveNewOwner()
-            }
-        }
+    /// Indicates whether to show owner onboarding flow.
+    @Published var shouldShowOwnerOnboarding = false
+    /// Indicates whether to show sitter onboarding flow.
+    @Published var shouldShowSitterOnboarding = false
+
+    /// Performs actions triggered by the owner role choice.
+    func owneRoleDidSelected() {
+        shouldShowOwnerOnboarding = !isCurrentOwnerExists
```

Comment:
typo
```suggestion
    func ownerRoleDidSelected() {
        shouldShowOwnerOnboarding = !isCurrentOwnerExists
```
We do the owner selection role in the View, not in view model based on existing code.